screenpy/resolutions/has_length.py

```diff
@@ -18,16 +19,21 @@ class HasLength:
         )
     """
 
+    @property
+    def length_to_log(self) -> str:
+        """Represent the length in a log-friendly way."""
+        return represent_prop(self.length)
```

Comment:
Make sure to add this method to the `exclude-members` list fo `HasLength` in `docs/api/resolutions.rst`!

Reply:
To be clear, should we be doing that with all the others as well?  None of the resolutions are currently excluding the `x_to_log` properties.  I would imagine we want to be consistent.

Reply:
Looking at the published docs, I don't see any of the `x_to_log` properties being displayed for the other resolutions.

Reply:
Oh, it's because none of the Resolutions have `:members:` directive, which is obvious to me now because we don't actually have any methods on resolutions (yet). 

I see you added those directives, but now i'm wondering—should we add the directives to exclude those members just in case we someday do add the `:members:` directive, or should we just assume Resolutions will never have methods on them?

Reply:
I had similar thought, but I wasn't sure.  Is that pattern going to fit every Resolution?

Reply:
I guess we can remove the `:exclude-members:` directives that you added. I didn't look before i asked for this; having just done a bunch of editing Actions documentation i just assumed we needed it.

I'd rather be consistent for now and deal with that when we need to.

Reply:
They are removed.